src/sgame/components/SpawnerComponent.cpp

```diff
@@ -46,6 +49,14 @@ void SpawnerComponent::Think(int timeDelta) {
 	Entity* blocker = GetBlocker();
 
 	if (blocker) {
+		if (!blocker->oldEnt) {
```

Comment:
We should make macros for this.

LOG_EVERY_INTERVAL()

and have another variant

LOG_EVERY_N().

Having this type of thing is super ugly for logs every time we want to do it.

Reply:
we can defer this change till later.

Reply:
@slipher does this mean we can merge that?